models/events/database.go

```diff
@@ -19,7 +19,6 @@ func (e *Event) BeforeUpdate(tc *gorm.DB) (err error) {
 }
 
 func isEventStatusSupported(event *Event) error {
-
 	if event.Status != Read && event.Status != Unread {
```

Comment:
Should this reference a struct/iota/enum from the Eventv2 schema?

Reply:
It is referencing, if you see the schema Status is defined as an enum